mdsa/pcoa.py

```diff
@@ -95,7 +95,8 @@ def pcoa(distance_matrix, algorithm, num_dimensions_out=10):
     eigenvectors, eigenvalues = algorithm.run(F_matrix, num_dimensions_out)
 
     # Ensure eigenvectors are normalized
-    eigenvectors = [vec / np.linalg.norm(vec) for vec in eigenvectors]
+    eigenvectors = np.array([vec / np.linalg.norm(vec)
```

Comment:
Ah, that didn't work. `np.linalg.norm(eigenvectors)` returns only a single value, never an array.